cob_twist_controller/include/cob_twist_controller/limiters/limiter_base.h

```diff
@@ -54,4 +77,27 @@ class LimiterBase
         const LimiterParams& limiter_params_;
 };
 
+/// Cartesian Base class for limiters, defining interface methods.
+class LimiterCartesianBase
+{
+    public:
+        explicit LimiterCartesianBase(const LimiterParams& limiter_params) : limiter_params_(limiter_params)
+        {}
```

Comment:
LimiterParams are inherited from LimiterBase class, thus the constructor should look like this:
```
explicit LimiterCartesianBase(const LimiterParams& limiter_params) : 
    LimiterBase(limiter_params)
{}
```